src/tests/test_point_redemption.py

```diff
@@ -179,24 +179,24 @@ def test_get_non_existing_redemption_requests_by_name(self):
                     headers=self.society_president,
                     content_type='application/json')
 
-        message = "does not exist"
+        message = "fetched successfully"
         response_details = json.loads(response.data)
 
         self.assertIn(message, response_details["message"])
-        self.assertEqual(response.status_code, 404)
+        self.assertEqual(response.status_code, 200)
+        self.assertFalse(response_details['data'])
```

Comment:
Why?

Reply:
This is a filter, 404 suggest the `url` is malformed which is not the case, I think it means the user passed the wrong data in the `url`.